### PR TITLE
fixed json ascii string error

### PR DESCRIPTION
**Pull request by Castronova**
bug fix that was causing resource download to fail in python 3.5.3.

### CODE REVIEW

File: hs_restclient/__init__.py [hunk 1]
```diff
@@ -789,7 +789,7 @@ def _getBagStream(self, pid, wait_for_bag_creation):
                 raise HydroShareHTTPException((bag_url, 'GET', r.status_code))
         elif r.headers['content-type'] == 'application/json':
             # this  is the case of bag being generated by hydroshare and not ready for download
-            content = json.loads(r.content)
+            content = json.loads(r.content.decode('utf-8'))
```
Comment (pkdash) [inline hs_restclient/__init__.py]: I assume this is a python 2 and 3 issue.

**Comment by pkdash** [comment]
@Castronova +1

**Comment by pkdash** [comment]
@Castronova Your PR is targeting master. It has to be develop. Do not merge into master.

**Comment by Castronova** [comment]
@pkdash Can we pull this in as a hot fix?

**Comment by pkdash** [comment]
@Castronova If we make it a hotfix do we also have to publish to PyPI? I think @aphelionz will be making a new release in a day or two. @aphelionz your thoughts on this?

File: hs_restclient/__init__.py (hunk 1 above)
Comment (aphelionz) [inline hs_restclient/__init__.py]: Surprisingly this was something that was going on between 3.5 and 3.6!

**Comment by aphelionz** [comment]
@pkdash I can do it basically as soon as i get the credentials

**Comment by pkdash** [comment]
@aphelionz It would be good if this change by @Castronova can be part of next release (1.2.7 )

**Comment by Castronova** [comment]
@pkdash I just want to update the jupyterhub image with this fix as soon as possible.  If release 1.2.7 will happen soon then that's fine with me...as long as there are non-breaking changes to the rest of the client.

**Pull request merged**